### DISCUSSION TITLE
Error model

### DISCUSSION THREAD

**Discussion by int3cd** [prisma/prisma#5907] [Q&A]
Hi, I executed the command: nix prisma introspect

Problem schema:
actstatus                                 Int?                                       @db.SmallInt

IDE Error:
<model block statement>, PrismaTokenType.DOUBLE_COMMENT, PrismaTokenType.ENTITY_NAME, PrismaTokenType.EOL, PrismaTokenType.TRIPLE_COMMENT or PrismaTokenType.} expected, got 'actstatus'

How can this be fixed?

**Comment by ryands17** [comment]
@int3cd 👋 

Could you share what was in your database when you ran this and also your `schema.prisma`?

**Reply by int3cd** [reply]
I can't show the whole scheme, only a part

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model addrobj {
  actstatus                                 Int?                                       @db.SmallInt
  aog                                    String                                     @id @db.VarChar(36)
  aoid                                      String?                                    @db.VarChar(36)
  aol                                   Int?                                       @db.SmallInt
  ...
}

If I delete @ db.SmallInt in the schema, then the validation goes to the next problem area @id @ db.VarChar (36)

**Reply by ryands17** [reply]
It seems that either you're using an older version of Prisma that doesn't support native types or your IDE doesn't support this syntax. Are you on the latest version of Prisma which is `2.17.0`? If not, could you update and check again.

Also Webstorm has some issues with Prisma so I would suggest using VSCode if you are seeing these errors in the editor.

Let me know if this still doesn't solve it.

**Reply by janpio** [reply]
What IDE are you actually using that is giving you this error message @int3cd? This doesn't look like any of the error messages of the tools we are offering. Or is this the Prisma VSCode extension somehow?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
